### ISSUE TITLE
wangning-homework

### ISSUE THREAD

**Issue by wnbupt** [BUPT-CAD/wora-app#10]
Name : wangning(王宁)
School: BUPT(北邮)
Platform : Mac
Data : Undetermined(待定)
Deadline:2017.7.2

**Comment by zixia** [comment]
No, As I told you in the class: the deadline must be a specific date. 

Please update.

**Comment by wnbupt** [comment]
My wora app link:
https://github.com/wnbupt/wora-app
Thank you for your advice~  @zixia

**Comment by huan** [comment]
Please link your demo screen record video in reply of this issue.

**Comment by wnbupt** [comment]
if you want to know more about wora-douban, the link is https://v.qq.com/x/page/r0517bmmm36.html
@zixia

**Comment by huan** [comment]
No.

The requirement of the homework is to submit the GIF format video in reply to this issue.

Please provide your demo screen record video in reply to this issue, in GIF format.

**Comment by wnbupt** [comment]
![wora-douban](https://user-images.githubusercontent.com/16443320/27506726-b7a1c92c-58f1-11e7-8522-4e7fbe2de7cf.gif)

**Comment by wnbupt** [comment]
I have updated this gif file in my github forked from  wora-app :)

**Comment by huan** [comment]
Great, good job, thanks.

**Comment by wnbupt** [comment]
Thank you !
